ko-search/movies-evan-likes/src/App/components/MovieList/MovieItem.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { centeredContainer } from 'sharedStyles.css';
+import { container, formatMovie, title, movieTitle, text } from './styles.css';
+
+export default props =>
+  <div className={(centeredContainer, formatMovie)} onClick={props.showEvanThoughts}>
+    <div className={(text, title)}>
+      <a onClick={props.handleTitleClick} className={movieTitle} href={props.movie.url}>{`${props
+        .movie.title}  | `}</a>
+      <strong>
+        {'Rating '}
+        {props.movie.rating}
+      </strong>
+      {' | '}
```

Comment:
`</strong> |
      {props.movie.year}`

becomes
`</strong>{' '}
      |
      {props.movie.year}`

which I preferred to format it like 
`{ ' | ' }`
 I think out of habit from using prettier I formatted the above one myself like that because I can't reproduce it but it would be better if I used a template string at least and made it one string rather than two separate ones.